test/e2e/smoke-test/packages/local-path-storage/0.0.20/local-path-storage-test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checking package is installed or not
+tanzu package installed list | grep "local-path-storage.community.tanzu.vmware.com"
+exitcode=$?
+
+if [ "${exitcode}" == 1 ] 
+then
+  version=$(tanzu package available list local-path-storage.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+  tanzu package install local-path-storage --package-name local-path-storage.community.tanzu.vmware.com --version "${version}"
+fi
+
+echo "Wating for Pods to reach ready state ..."
+sleep 15s
```

Comment:
@aman556 
Maybe we could use `kubectl wait` here, like this - https://github.com/vmware-tanzu/community-edition/blob/7da68b52bf1551bad807113c402810f3761eb167/test/aws/deploy-tce-managed.sh#L75

The benefit is, we won't wait for a constant X time always, instead we would wait just enough time where the pod is ready and then move on. We would still have a max timeout time for which we will wait. Let's have 30s timeout time too, that's okay

Also, this is all assuming that `tanzu package install` does Not wait for pods to be up and running and ready, which I gotta check